package.json

```diff
@@ -15,6 +15,7 @@
   "scripts": {
     "format": "wp-scripts format ./cli/**/*.mjs",
     "lint": "wp-scripts lint-js ./cli --ext mjs",
-    "research": "./cli/run.mjs"
+    "research": "./cli/run.mjs",
+    "benchmark": "npm run research --silent -- benchmark-url"
```

Comment:
Please remove this for now, other commands don't have this shorthand either. What would be great instead though is to document the benchmark command in `cli/README.md`, similar to how the other commands are documented there.

Reply:
Removed and updated the readme file.